test/infrastructure/docker/config/webhook/manifests.yaml

```diff
@@ -1,122 +0,0 @@
----
```

Comment:
I think you have to create this file again. make generate-manifests-docker-infrastructure should generate it